src/main.cpp

```diff
@@ -182,7 +182,7 @@ int main(int argc, char** argv)
                 "\t -j, --injection-timeout <seconds>\n"
                 "\t                           Injection timeout (in seconds, 0 == no timeout)\n"
                 "\t -t <timeout>              Timeout (in seconds)\n"
-                "\t -o <format>               Output format (default or csv)\n"
+                "\t -o <format>               Output format (default, csv, or json)\n"
```

Comment:
We seem to have missed adding `kv` previously to the list, please do add that too.